admin-dev/themes/new-theme/js/components/multistore-header.js

```diff
@@ -72,6 +73,38 @@ const initMultistoreHeader = () => {
   });
 };
 
+const initContextualNotification = () => {
+  const configKeyShopPrefix = 'missing-color-shop-';
+  const configKeyGroupPrefix = 'missing-color-group-';
+  const multistoreHeader = document.querySelector('.header-multishop');
+  const notificationMessage = multistoreHeader.dataset.headerColorNotification;
+
+  if (notificationMessage.length <= 0) {
+    return;
+  }
+
+  // make localstorage key for this context
+  const contextualNotification = new ContextualNotification();
+  let notificationKey = configKeyGroupPrefix + multistoreHeader.dataset.groupId;
+
+  if (multistoreHeader.hasAttribute('data-shop-id')) {
+    notificationKey = configKeyShopPrefix + multistoreHeader.dataset.shopId;
+  }
+
+  // check if key exists, if yes: display or not depending on given value
```

Comment:
this code looks like the code for the other notification, but since it's specific to the fact that here we deal with group and shop contexts, I didn't want to deal with this in the `contextual-notification` component, whose goal is to be reusable, even for notifications not related to context.